rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -301,7 +301,8 @@ bool gs_frame::shown()
 
 void gs_frame::hide()
 {
-	Emu.CallAfter([this]() {QWindow::hide(); });
+	m_flip_showed_frame = false;
```

Comment:
Why is this done here?
Seems like a remnant of old code

Reply:
Fixed.